src/configimpl.l

```diff
@@ -3,8 +3,8 @@
  * Copyright (C) 1997-2015 by Dimitri van Heesch.
  *
  * Permission to use, copy, modify, and distribute this software and its
- * documentation under the terms of the GNU General Public License is hereby 
- * granted. No representations are made about the suitability of this software 
+ * documentation under the terms of the GNU General Public License is hereby
+ * granted. No representations are made about the suitability of this software
```

Comment:
The review is at these 2 lines, but accounts for the entire file.
I do agree that the spacing should be consistent in the files, at the moment there is a mixture of tabs and spaces (especially in the lex files), but I don't think this should be "hidden" in other pull requests (and thus hiding real relevant changes for the pul request for easy review), I think it would be better to have a separate pull request for it.

Reply:
Ok, I can easily move the Commit to its own PR. Will do so in the next days (have to spend some time on my day-job. :-) )